content/docs/querying/operators.md

```diff
@@ -25,16 +31,18 @@ value of every data sample in the vector. E.g. if a time series instant vector
 is multiplied by 2, the result is another vector in which every sample value of
 the original vector is multiplied by 2.
 
-**Between two instant vectors**, a binary arithmetic operator only applies to
-vector elements that have identical sets of labels between the two vectors.
-Vector elements that don't find an exact label match on the other side get
-dropped from the result. The metric name of the result vector is carried over
-from the left hand side of the expression.
+**Between two instant vectors**, a binary arithmetic operator is applied to
+each entry in the left-hand-side vector and its [matching element](#vector-matching)
+on the right hand vector. The result is propagated into the result vector and the metric
```

Comment:
I think I had a comment here: it's either "in the right hand vector" or "on the right hand [side]", but not "on the right hand vector".